src/app/orders/container.rs

```diff
@@ -82,7 +76,8 @@ impl<Ms: 'static, Mdl, INodes: IntoNodes<Ms> + 'static, GMs: 'static> Orders<Ms,
         );
 
         let cmd = cmd.map(move |msg| app.mailbox().send(handler(msg)));
-        CmdManager::perform_cmd(cmd);
+        // @TODO_B: CmdManager, which we are trying to get rid of, previously wraped this function
```

Comment:
Not so sure about what to do with this call to `spawn_local`

Reply:
Why we want to remove `CmdManager`?

Reply:
This TODO was entered in error.